app/konnect/gateway-manager/deploy-service.md

```diff
@@ -2,17 +2,15 @@
 title: Proxy and Test a Service
```

Comment:
Should we just remove this page completely since it contains pretty much the same info as the Add your API page in the getting started section? I think I'd only keep it if we were worried that people would be looking in this section for something about adding a service. But I'm guessing most people use the search bar or Google.